packages/ruleset/src/ibm-oas.js

```diff
@@ -6,6 +6,14 @@
 const { oas3 } = require('@stoplight/spectral-formats');
 const { oas } = require('@stoplight/spectral-rulesets');
 const ibmRules = require('./rules');
+
+// Spectral's "no-$ref-siblings" rule is configured to run on
+// OpenAPI 3.0.x documents (ref sibling attributes are allowed in OpenAPI 3.1.x).
+// However, we want to enable this rule also for OpenAPI 3.1.x documents,
+// so we'll just tweak Spectral's rule definition here.
+const noRefSiblings = oas.rules['no-$ref-siblings'];
+noRefSiblings.formats = [oas3];
```

Comment:
You could do this without defining a variable, right?

```suggestion
oas.rules['no-$ref-siblings'].formats = [oas3];
```

No big deal either way, just checking if that was defined for any particular reason

Reply:
Yes, you're right.  Initially, I was using that variable a little later but then backed that change out.  I'll change this.